pkgs/applications/networking/browsers/vivaldi/default.nix

```
@@ -20,11 +20,11 @@ let
   vivaldiName = if isSnapshot then "vivaldi-snapshot" else "vivaldi";
 in stdenv.mkDerivation rec {
   pname = "vivaldi";
-  version = "5.4.2753.37-1";
+  version = "5.4.2753.47-1";
```

```suggestion
  version = "5.4.2753.47";
```
Isn't this just the deb version number that is unused by upstream?

---

It does appear to be unused, if the past PRs are anything to go by. I'll fix the update script.